pkg/chunk/aws/metrics.go

```diff
@@ -0,0 +1,60 @@
+package aws
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/weaveworks/common/instrument"
+)
+
+type metrics struct {
```

Comment:
I would call it `dynamoDBMetrics` (and filename + `newMetric()` renamed accordingly) considering in this package we also have the S3 client and these metrics are DynamoDB specific.